packages/forms/src/components/FloatingSearchButtonIcon/index.js

```diff
@@ -0,0 +1,143 @@
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { ButtonIcon } from 'react-rainbow-components';
+import FloatingSearch from '../FloatingSearch';
+
+const searchStyle = {
+    maxWidth: '500px',
+};
+
+const FloatingSearchButtonIcon = (props) => {
+    const {
+        assistiveText,
+        className,
+        disabled,
+        icon,
+        id,
+        shaded,
+        size,
+        style,
+        tabIndex,
+        title,
+        type,
```

Comment:
type does not make sense here since it will be type="button" always, there are no use cases for submit and reset